code/datums/quirks/negative_quirks.dm

```diff
@@ -530,9 +530,11 @@
 	icon = "tg-prosthetic-leg"
 	value = -3
 	var/slot_string = "limb"
+	/// the original limb from before the prosthetic was applied
+	var/obj/item/bodypart/old_limb
```

Comment:
new vars should go after vars declared on parent, can you re-arrange this since you're adding a new var in this PR?